index.ts

```diff
@@ -1,5 +1,5 @@
-import * as chromeFinder from 'chrome-launcher/dist/src/chrome-finder';
-import * as chromeUtils from 'chrome-launcher/dist/src/utils';
+import * as chromeFinder from 'chrome-launcher/dist/chrome-finder';
```

Comment:
I see these paths changed, but I don't see any changes that indicate these are now being built to different folders. Am I missing something?

Reply:
![Screen Shot 2019-09-11 at 11 28 07 AM](https://user-images.githubusercontent.com/4612419/64711216-42eccf00-d487-11e9-9acb-cefcf056c00c.png)